scalyr_agent/configuration.py

```diff
@@ -1497,6 +1499,41 @@ def api_key(self):
         """Returns the configuration value for 'api_key'."""
         return self.__get_config().get_string("api_key")
 
+    @property
+    def server_url(self):
+        """Returns the configuration value for 'server_url'."""
+        return self.__get_config().get_string("server_url")
+
+    @property
+    def auth(self):
+        """Returns the configuration value for 'auth'."""
+        return self.__get_config().get_string("auth")
+
+    @property
+    def client_id(self):
```

Comment:
Please move all oauth2 configuration keys under a new oauth2 key:
oauth2:
->> client_id:
->> client_secret:
  ....

Reply:
I went with `oauth_` prefix instead which goes with the current pattern used for kubernetes keys (`k8s_`)